<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Vorbereitung (Bitte ausfüllen!)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Koordinaten Eingabe:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="bChromePOS">
                  <property name="text">
                   <string>Chrome geladen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lcolorChrome">
                  <property name="text">
                   <string>RGB</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="bGroePOS">
                <property name="text">
                 <string>Größe</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bWarenkorbPOS">
                <property name="text">
                 <string>in Warenkorb</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bKassePOS">
                <property name="text">
                 <string>zur Kasse</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QPushButton" name="bZahlungGelPOS">
                  <property name="text">
                   <string>Zahlungsmenue geladen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lcolorZahl">
                  <property name="text">
                   <string>RGB</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="bArtDerZPOS">
                <property name="text">
                 <string>Art der Zahlung</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QPushButton" name="bPPgelPOS">
                  <property name="text">
                   <string>Paypal geladen</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lcolorPP">
                  <property name="text">
                   <string>RGB</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="bPPFeldPOS">
                <property name="text">
                 <string>Paypal Feld</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="lChromePOS">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lGroePOS">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lWarenkPOS">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lKassePOS">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lZahlungGelPOS">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lArtZahlPOS">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lPPgelPOS">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="lPPfeldPOS">
                <property name="text">
                 <string>TextLabel</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;b&gt;Produkt:&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="lineName"/>
              </item>
              <item>
               <widget class="QPushButton" name="bName">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QComboBox" name="comboKategorie"/>
              </item>
              <item>
               <widget class="QPushButton" name="bKategorie">
                <property name="text">
                 <string>Kategorie</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QComboBox" name="comboZahlung"/>
              </item>
              <item>
               <widget class="QPushButton" name="bZahlungsart">
                <property name="text">
                 <string>Zahlungsart</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QComboBox" name="comboGroesse"/>
              </item>
              <item>
               <widget class="QPushButton" name="bGroesse">
                <property name="text">
                 <string>Größe</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLineEdit" name="linePPlogin"/>
              </item>
              <item>
               <widget class="QPushButton" name="bPaypallogin">
                <property name="text">
                 <string>Paypal Login / Kreditkartennummer</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLineEdit" name="linePPPasswort">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="bPaypalPasswort">
                <property name="text">
                 <string>PayPal Passwort / CVV-code</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QComboBox" name="lineMonat"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="text">
                 <string>Kartenablauf Monat</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QComboBox" name="lineJahr"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Kartenablauf Jahr</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLineEdit" name="lineChromepath"/>
              </item>
              <item>
               <widget class="QPushButton" name="bChromepath">
                <property name="text">
                 <string>Chrome Pfad</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Bitte den Pfad zur .exe Datei von Chrome kopieren und die '\'  durch '/' ersetzen!</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>.</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QPushButton" name="bSpeichern">
        <property name="text">
         <string>Speichern</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CheckinPPeingeloggt">
        <property name="text">
         <string>schon in Paypal 
eingeloggt?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Profelauf">
        <property name="text">
         <string>Probelauf 
(ohne Zahlung)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
